src/GitHub.Exports/Services/MetricsService.cs

```diff
@@ -137,11 +121,25 @@ static StringContent SerializeRequest(UsageModel model)
                 {
                     dict.Add(ToJsonPropertyName(prop.Name), prop.GetValue(model));
                 }
+                else
+                {
+                    var value = prop.GetValue(model);
+
+                    if (value == null)
+                    {
+                        dict.Add(ToJsonPropertyName(prop.Name), value);
+                    }
+                    else
+                    {
+                        dict.Add(ToJsonPropertyName(prop.Name), ToModelDictionary(value));
```

Comment:
Are we expecting nested dictionaries?

Reply:
Not really, I just undid your changes to only output strings, this is the code that was here before, but I think I forgot to revert the other change where you call this method twice - it should be `ToModelDictionary(model)`

Reply:
No, you caught that change